### ISSUE TITLE
May add multi_head support ?

### ISSUE THREAD

**Issue by svjack** [tensorflow/adanet#33]
For multi-objective learning, may use multi_head to define "logits", head used in adanet
to increment network construction,.
Can I used multi_head  in adanet, or the support may be added in the future ?

**Comment by cweill** [comment]
@svjack: We don't currently support multi-head. This has been a top feature request now, so I'm looking into adding it.
